libs/employees/src/lib/employees/employees.component.html

```diff
@@ -1,7 +1,7 @@
 <h1>Employees</h1>
 
 <razroo-fully-architected-dashboard-data-table
-  [tableData]="employees$ | async"
+  [tableData]="employees$| async"
```

Comment:
Keep the space between the employees$ and |

Reply:
Fixed